R/lbl_helpers.r

```diff
@@ -399,10 +399,42 @@ as_lbl_function <- function(x, env = caller_env()) {
 
 # Adapted from rlang:::abort_coercion
 abort_coercion_function <- function(x) {
-  x_type <- rlang::friendly_type_of(x)
+  x_type <- friendly_type_of(x)
```

Comment:
Ha, sorry for the bad advice before. This looks good to me.

Reply:
Not bad advice, just incomplete, as it turns out.